test/index.js

```diff
@@ -151,5 +153,11 @@ describe('Apostrophe-multisite', function() {
       });
       expect(response).to.have.property('statusCode', 302);
     });
+
+    // it('set up why-is-node-running', function() {
+    //   setTimeout(function () {
+    //     running(); // logs out active handles that are keeping node running
+    //   }, 60000);
+    // });
```

Comment:
Need to remove these comments?

Reply:
No, it is there to help with any future recurrence of issues with apos.destroy not behaving.